app/src/main/java/edu/wwu/avilatstudents/journey/MainActivity.java

```diff
@@ -182,14 +186,18 @@ public void onClick(View view) {
             @Override
             public void onClick(View view) {
                 sessionManager.logout();
+                dbm.signOut(sessionManager.getEmail(), sessionManager.getAuthentication());
                 actionBar.show();
                 // reset nav buttons to base color (selected color applied appropriately in switch)
                 setNavButtonColorSelected(journeysFab);
                 updateActionBar(R.id.journeys_fab);
                 visibleLayout.setVisibility(View.GONE);
                 journeysLayout.setVisibility(View.VISIBLE);
                 visibleLayout = journeysLayout;
-                visibleLayout.requestFocus();
+
+                Intent gotoLogin = new Intent(MainActivity.this, LoginActivity.class);
```

Comment:
sessionManager.logout() starts the Login Activity already, so there is no need for this.